padlockcloud/server.go

```diff
@@ -98,7 +100,7 @@ type ServerConfig struct {
 }
 
 // The Server type holds all the contextual data and logic used for running a Padlock Cloud instances
-// Users should use the `NewServer` function to instantiate an `Server` instance
+// Users should use the `NewServer` function to instantiate an `Server`f instance
```

Comment:
I'm guessing this was accidental?

Reply:
Yep, just a slight mistake I saw when I was reading over the commits before the PR.

Reply:
Just a reminder to clean this up ;)